.github/workflows/update_image.yml

```diff
@@ -29,3 +32,59 @@ jobs:
         run: docker build -t $image_tag .
       - name: Publish Tag
         run: docker push $image_tag
+
+  update_docs:
+    needs: update_image
+    runs-on: ubuntu-20.04
+    container: osgeo/gdal:ubuntu-small-3.6.1
+
+    steps:
+    - uses: actions/checkout@v2
```

Comment:
If we are going through the and doing maintenance, should we update `checkout@v2` to `v3`?

Reply:
https://github.com/actions/checkout